src/mixs/schema/mixs.yaml

```diff
@@ -25,6 +25,99 @@ subsets:
   nucleic acid sequence source:
   investigation:
 enums:
+  CurLandUseEnum:
+    permissible_values:
+      badlands:
```

Comment:
Can we add meanings to some of these PV's?

For example, is this an appropriate meaning for the badlands PV? http://purl.obolibrary.org/obo/ENVO_00000127